utils/switch_task_to_static/docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+services:
+  generate_chunks_for_task:
+    image: cvat/server:v2.45.0
+    environment:
+      CVAT_POSTGRES_HOST: cvat_db
+    volumes:
+      - cvat_data:/home/django/data
+      - ./switch_task_to_static_cache.py:/home/django/switch_task_to_static_cache.py:ro
```

Comment:
```suggestion
```

This line doesn't work with the command from README.md. I think it's better to just use the same approach as we have in dataset_manifest instead - just use the file in `utils/switch_task_to_static/`. This will require https://github.com/cvat-ai/cvat/pull/9552/files#r2394268694.